HotelBooking/Views/book_reservation_view.py

```diff
@@ -4,6 +4,15 @@
 from typing import Tuple, List
 from PyInquirer import prompt
 from HotelBooking.Models.room import ROOM_TYPE
+from datetime import datetime
+
+def ValidateDate(date_text):
```

Comment:
can you move this function to Views/utils.py? 
Also, snake case.

Reply:
Refactor it to Utils, so the same validation method can be shared by all other views.

Reply:
yessss